docker-compose.yml

```diff
@@ -5,10 +5,10 @@ version: '3.7'
 
 
 x-shared: &shared
```

Comment:
This is the only file I'm not sure about, will need to be tested in deployment.

Reply:
When I run the new version of this file and the old version of it through http://www.yamllint.com/ they're fundamentally different - some of the indentation changes has actually changed the structure it generates.

For example the environment for x-shared only has one field with the new setup but had many with the old one. Seems to have lost that nested structure.

Recommend we revert before merge.